Stanford/JumpstartLab/Install/Block/BlockSettings.php

```diff
@@ -139,6 +139,19 @@ public function execute(&$args = array()) {
       $update->execute();
     }
 
+    $disable = array(
```

Comment:
This functionality is in https://github.com/SU-SWS/stanford_install_tasks/blob/7.x-1.x/Stanford/Jumpstart/Install/Block/BlockSettings.php 

It looks like it might be called too early as the modules may not yet be enabled. Can you revise and refactor the existing task so we don't have the duplication? Thanks.

It looks like the profile is calling this step